src/utils/helpers/list/buildQueryFilters.ts

```diff
@@ -15,8 +15,6 @@ export default function buildQueryFilters(
     if (!filterDefinition) return filterObj
     return { ...filterObj, ...constructFilter(filterDefinition, filterName, filterValue) }
   }, {})
-  // If no filters, return a dummy filter to prevent GraphQL empty object error
-  if (Object.keys(graphQLfilter).length === 0) return { templateCode: { isNull: false } }
```

Comment:
This is the bit of ugliness that can be removed as a result of [this change](https://github.com/openmsupply/conforma-server/pull/865#discussion_r977178230) in the back-end.